### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by bogem**
Delete some redundant code

### CODE REVIEW

**Comment by bogem** [comment]
Any questions or offers?

**Comment by bogem** [comment]
Any questions or offers?

**Comment by bogem** [comment]
So?

File: string_slice.go [hunk 1]
```diff
@@ -39,7 +36,7 @@ func writeAsCSV(vals []string) (string, error) {
 		return "", err
 	}
 	w.Flush()
-	return strings.TrimSuffix(b.String(), fmt.Sprintln()), nil
+	return strings.TrimSuffix(b.String(), "\n"), nil
```
Comment (moorereason) [inline string_slice.go]: This line doesn't change functionality on any platform.

**Pull request merged**